mmv1/templates/terraform/examples/clouddeploy_custom_target_type_gcb_repo_skaffold_modules.tf.erb

```diff
@@ -0,0 +1,17 @@
+resource "google_clouddeploy_custom_target_type" "<%= ctx[:primary_resource_id] %>" {
+    location = "us-central1"
+    name = "<%= ctx[:vars]['custom_target_type_name'] %>"
+    description = "My custom target type"
+    custom_actions {
+      render_action = "renderAction"
+      deploy_action = "deployAction"
+      include_skaffold_modules {
+        configs = ["my-config"]
+        google_cloud_build_repo {
+            repository = "projects/777/locations/us-central1/connections/git/repositories/example"
```

Comment:
only request I have is to update the value so that it doesn't look like placeholder value. This is only for examples since this will be shown in docs.

Reply:
Done.